src/stores/widgets/RegionWidgetStore.ts

```diff
@@ -1,6 +1,7 @@
 import {action, observable, computed, makeObservable} from "mobx";
 import {IOptionProps} from "@blueprintjs/core";
 import {AppStore, FrameStore, RegionStore} from "..";
+import {CARTA} from "carta-protobuf";
```

Comment:
carta-protobuf import after third-party and before local imports

Reply:
done.